go.mod

```diff
@@ -489,3 +489,5 @@ replace (
 exclude k8s.io/client-go v12.0.0+incompatible
 
 exclude github.com/openshift/client-go v3.9.0+incompatible
+
+replace github.com/AdamKorcz/go-fuzz-headers-1 v0.0.0-20230329111138-12e09aba5ebd => github.com/AdamKorcz/go-fuzz-headers-1 v0.0.0-20230919221257-8b5d3ce2d11d
```

Comment:
I'd prefer to have as little replaces as possible. We already have a replace for cosign that introduces rektor that depends on this lib and has upgraded some time ago. Could we just upgrade our fork of cosign? cc @dhaus67 

- https://github.com/sigstore/rekor/commit/24cb647106d74051ba70ce867b5c7319ad29e3b9

```
go mod why github.com/AdamKorcz/go-fuzz-headers-1
# github.com/AdamKorcz/go-fuzz-headers-1
github.com/stackrox/rox/pkg/signatures
github.com/sigstore/cosign/v2/pkg/cosign
github.com/sigstore/rekor/pkg/types/hashedrekord/v0.0.1
github.com/sigstore/rekor/pkg/types/hashedrekord/v0.0.1.test
github.com/AdamKorcz/go-fuzz-headers-1
```

Reply:
Sure! I'd be happy to close this PR if we can instead update our fork!